mongorestore/oplog.go

```diff
@@ -110,6 +110,15 @@ func (restore *MongoRestore) RestoreOplog() error {
 			//skip no-ops
 			continue
 		}
+
+		if restore.serverVersion.GTE(db.Version{4, 4, 0}) {
+			entryName := entryAsOplog.Object[0].Key
```

Comment:
I am not sure if this is risky always assume `Object` field is not empty. Maybe worth checking it first and add extra condition _entryAsOplog.Operation == "c"_ ? What do you think @tfogo

Reply:
Yeah, I agree. a) check it's a command and b) to be defensive, check that the length of entryAsOplog.Object != 0.